CHANGELOG.md

```diff
@@ -15,6 +15,11 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 - Removes our proprietary protocol in favor of unifying on the Language Server Protocol. Using the standard protocol lets us support more editors with less code.
 - Notices changes to polymer.json immediately. Currently this is useful for changing lint rules without reloading your editor.
 - Supports finding all references for elements. Finds references to the element in HTML tags.
+- Supports finding definitions and all references of CSS Custom Properties.
+  - When finding definitions of the property it will return all places where
+    the property is assigned to.
+  - When finding all references it will find all places where the property
+    is read with `var(--foo)` or `@apply(--bar)` syntax.
```

Comment:
The apply should be without parentheses ;)

Reply:
Does it show that I've been testing out `--fix` a lot the past few weeks?